bokehjs/src/coffee/common/embed.coffee

```diff
@@ -163,4 +179,4 @@ embed_items = (docs_json, render_items, websocket_url) ->
 
 module.exports =
   embed_items: embed_items
-  inject_css: inject_css
+  inject_css: inject_css
```

Comment:
your editor seems to hate trailing newline but emacs/vi/traditional editors really want it to be there :-)